.github/workflows/setup-gcloud-it.yml

```diff
@@ -69,3 +69,22 @@ jobs:
       - name: Verify Exported
         shell: bash
         run: test -f $GOOGLE_APPLICATION_CREDENTIALS
+
+  project_id:
+    name: setup-gcloud with configured project
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: setup-gcloud
+        uses: ./setup-gcloud/
+        with:
+          project_id: ${{ secrets.SETUP_GCLOUD_IT_PROJECT_ID }}
```

Comment:
Still need to specify the GCP SA key to use.

Reply:
The project setting happens before authentication, so you don't actually need the service account here. It just skips service account activation if one is not given. We do need to create the `SETUP_GCLOUD_IT_PROJECT_ID` secret though to get the tests to pass.

Reply:
Ack, thanks for clarifying.